test/Syntax/round_trip_parse_gen.swift

```diff
@@ -599,8 +599,17 @@ func foo() {}
 ##"abc \##(foo)"##
 
 foo()
-#if true
+#if COND1
   .bar?()!
+#elseif COND2
+  #if true
+    .call()
+  #else
+    #if true
```

Comment:
Would it be worth having another case where there is an intervening member access expression in the middle of two clauses, to check that the base is present in that nesting scenario? For example,

```swift
foo()
#if COND1
  .bar?()!
#elseif COND2
  #if true
    .call()
  #else
    .somethingElse
    #if true
      ...
```